airbyte-webapp/src/packages/cloud/views/auth/OAuthLogin/OAuthLogin.module.scss

```diff
@@ -43,7 +43,7 @@
 }
 
 .github {
-  background: #333;
+  background: colors.$githubColor;
```

Comment:
I think it would make sense keeping that color in this file instead of extracting it to _colors, since it should not be used anywhere else, and instead use a `/* stylelint-disable-next-line color-no-hex */` here to disable the check for this line.

Reply:
Ok, got it, will fix